cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -566,6 +573,164 @@ public void pullPackage(String org, String name, String version, Path packagePat
         }
     }
 
+    /**
+     * Pull a tool from central.
+     *
+     * @param toolId                    The id of the tool.
+     * @param version                   The version of the package.
+     * @param balaCacheDirPath    The package path in Bala cache.
+     * @param supportedPlatform         The supported platform.
+     * @param ballerinaVersion          The ballerina version.
+     * @param isBuild                   If build option is enabled or not.
+     * @return An array containing the organization, package name and version.
+     * @throws CentralClientException   Central Client exception.
+     */
+    public String[] pullTool(String toolId, String version, Path balaCacheDirPath, String supportedPlatform,
+                         String ballerinaVersion, boolean isBuild) throws CentralClientException {
+        String resourceUrl = "/" + TOOLS + "/" + toolId;
+        boolean enableOutputStream =
+                Boolean.parseBoolean(System.getProperty(CentralClientConstants.ENABLE_OUTPUT_STREAM));
+        String toolSignature = toolId;
+
+        // TODO: remove once mocking is done
+        String url = "https://09edc8a7-fa97-48d7-b07a-b7709ce6101d.mock.pstmn.io" + resourceUrl;
```

Comment:
@gayaldassanayake What's the purpose of this?

Reply:
We haven't recieved the central APIs. So I'm using a mock URL. Need to replace with the central URL once recieved.

Reply:
Ack